AUTHORS.rst

```diff
@@ -126,3 +126,4 @@ their individual contributions.
 * `Wilfred Hughes <https://www.github.com/wilfred>`_
 * `Zac Hatfield-Dodds <https://www.github.com/Zac-HD>`_ (zac.hatfield.dodds@gmail.com)
 * `Zebulun Arendsee <https://www.github.com/arendsee>`_ (zbwrnz@gmail.com)
+* `Afrida Tabassum <https://github.com/oxfordhalfblood>`_ (afrida@gmail.com)
```

Comment:
This list is alphabetised, so you should probably be closer to the top :wink:

Reply:
Awesome!